src/NodeAnalyzer/MethodArgumentAnalyzer.php

```diff
@@ -26,18 +27,36 @@ public function hasMethodSecondArgument(Node $node): bool
 
         if (count($node->args) < 2) {
             return false;
-        }
+    }
+
+return true;
```

Comment:
There're some CS issues, a simple `composer fix-cs` should fix :smiley:

Reply:
Thanks @carusogabriel. I try.